project-koku/koku
# COST-1021: GCP instance endpoint blows up with bad delta.

It looks like I forgot to go back and update the gcp serializer for delta validation to allow usage as a delta option for the storage and instance-types api. The ground work to allow for usage delta use in the provider map has already been added, so I just need to enable it in the serializer.

----- 
How to Verify:
Please bring up the codebase and run `make load-test-customer-data` then check the following endpoints.

Now Valid Deltas:
- http://localhost:8000/api/cost-management/v1/reports/gcp/instance-types/?filter[resolution]=daily&filter[time_scope_value]=-1&filter[time_scope_units]=month&delta=usage
- http://localhost:8000/api/cost-management/v1/reports/gcp/storage/?filter[resolution]=daily&filter[time_scope_value]=-1&filter[time_scope_units]=month&delta=usage
- http://localhost:8000/api/cost-management/v1/reports/gcp/cost/?filter[resolution]=daily&filter[time_scope_value]=-1&filter[time_scope_units]=month&delta=cost

Now Invalid Deltas: 
- http://localhost:8000/api/cost-management/v1/reports/gcp/instance-types/?filter[resolution]=daily&filter[time_scope_value]=-1&filter[time_scope_units]=month&delta=cost
- http://localhost:8000/api/cost-management/v1/reports/gcp/storage/?filter[resolution]=daily&filter[time_scope_value]=-1&filter[time_scope_units]=month&delta=cost
- http://localhost:8000/api/cost-management/v1/reports/gcp/cost/?filter[resolution]=daily&filter[time_scope_value]=-1&filter[time_scope_units]=month&delta=usage
